app/views/layouts/_sidebar_navigation.html.erb

```diff
@@ -82,6 +82,16 @@
   </li>
 <% end %>
 
+<!-- Nav Item - Surveys -->
+<% if visible_in_role?("role_survey") %>
+  <li class="nav-item <%= 'active' if params[:controller] == 'surveys' %>">
+    <a class="nav-link" href="/surveys">
+      <i class="fas fa-fw fa-walking"></i>
```

Comment:
TODO: correct icon is already requested per slack

Reply:
Philipp Wilimzig:

> chart-bar

Reply:
![Bildschirmfoto 2021-06-17 um 18 39 37](https://user-images.githubusercontent.com/1942953/122438802-62d5ee80-cf9b-11eb-8f01-fd6ce9759f75.png)